plugins/wpe-headless/composer.json

```diff
@@ -7,15 +7,17 @@
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
         "php-parallel-lint/php-parallel-lint": "^1.2",
         "phpcompatibility/phpcompatibility-wp": "^2.1",
+        "phpunit/phpunit": "~6",
```

Comment:
Are there any issues with running a later version?

Reply:
I stand somewhat correct. WordPress recommends phpunit `7.x` for now. https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/#setup

Phpunit recommends a different phpunit version for each version of php. https://phpunit.de/getting-started-with-phpunit.html

My guess is they immediately use new features.

I'll update to `~7`